### DISCUSSION TITLE
brew不支持macos ventura版本

### DISCUSSION THREAD

**Discussion by lsy-73** [Homebrew/discussions#3843] [Getting started] [answered]
#我的提示如下：
Warning: Git could not be found in your PATH.
Homebrew uses Git for several internal functions, and some formulae use Git
checkouts instead of stable tarballs. You may want to install Git:
  brew install git

Warning: No developer tools installed.
Install the Command Line Tools:
  xcode-select --install

Warning: You are using macOS 13.
We do not provide support for this pre-release version.
You will encounter build failures with some formulae.
Please create pull requests instead of asking for help on Homebrew's GitHub,
Twitter or any other official channels. You are responsible for resolving
any issues you experience while you are running this
pre-release version.

Warning: Ruby version 2.6.10 is unsupported on macOS 13. Homebrew
is developed and tested on Ruby 2.6.9, and may not work correctly
on other Rubies. Patches are accepted as long as they don't cause breakage
on supported Rubies.

**Comment by carlocab** [comment, marked as answer]
Please install the latest "Command Line Tools for Xcode 14.1" from https://developer.apple.com/download/all/. At the time of writing, this is "Command Line Tools for Xcode 14.1 Release Candidate 2". You'll need to register an account with Apple, but the download is free.

That should allow you to run `brew update`.

See also #3822.

**Reply by lsy-73** [reply]
确实是这个问题，解决了，谢谢

**Reply by ye808** [reply]
mac os14能不能这样解决

**Reply by ye808** [reply]
还是不能😩
